tests/darcy_velocity.prm

```diff
@@ -0,0 +1,148 @@
+# blob should be advected horizontally with the solid and
+# vertically as it rises up due to its buoyancy.
+
+############### Global parameters
+
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 5000
+set Use years in output instead of seconds = true
+set Adiabatic surface temperature          = 1600
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Max nonlinear iterations               = 5
+set CFL number                             = 0.2
+set Output directory                       = darcy_velocity
+set Pressure normalization                 = surface
+set Surface pressure                       = 0
+set Maximum time step                      = 1000
+
+############### Parameters describing the model
+# Let us here choose again a box domain
+# where we fix the temperature at the bottom and top,
+# allow free slip along the boundaries
+# and include melt migration.
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 200000
+    set Y extent = 100000
+    set X repetitions = 2
+  end
+end
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = 2, 3
+end
+
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = left x:function, right x:function
+  set Tangential velocity boundary indicators = top, bottom
+  subsection Function
+    set Coordinate system = cartesian
+    set Variable names = x,y,t
+    set Function expression = 1;0
+  end
+end
+
+############### Compositional fields
+# We want to use three compositional fields, the porosity and
+# and two additional fields. The solid field will be advected with
+# the solid speed, and the fluid field will be advected as the sum of
+# the solid velocity and the vertical buoyancy the fluid has. The
+# porosity field is uniform to allow for a uniform fluid velocity.
+subsection Compositional fields
+  set Number of fields = 3
+  set Names of fields = porosity, solid, fluid
+  set Compositional field methods = darcy field, field, darcy field
+  set Types of fields = porosity, generic, generic
+end
+
+subsection Boundary composition model
+  set List of model names = initial composition
+  set Fixed composition boundary indicators = bottom, top, left, right
+end
+
+subsection Boundary temperature model
+  set List of model names = box
+
+  subsection Box
+    set Bottom temperature = 293
+    set Top temperature    = 293
+  end
+end
+
+
+# We then choose a vertical gravity model and describe the
+# initial temperature with a vertical gradient.
+subsection Gravity model
+  set Model name = vertical
+  subsection Vertical
+    set Magnitude = 10.0
+  end
+end
+
+
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression       = 293
+  end
+end
+
+subsection Initial composition model
```

Comment:
Can you add a few briefs line of documentation saying something to the effect of "The initial porosity is constant, while the solid and fluid" fields are circles of size X/Y in the center of the model domain"?